src/Tools/dotnet-monitor/Exceptions/ExceptionsPostConfigureOptions.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Diagnostics.Monitoring.Options;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Diagnostics.Tools.Monitor.Stacks
+{
+    internal sealed class ExceptionsPostConfigureOptions : IPostConfigureOptions<ExceptionsOptions>
```

Comment:
Perhaps we can have a baseclass for this since the post configure options look almost identical between features?